site/resources.md

```diff
@@ -20,17 +20,17 @@ Resources
 
   _add description_
 
-* Siphon (remote_open)
+* [Siphon (remote_open)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/dataAccess/siphon-RemoteOpen.ipynb)
```

Comment:
Two quick things....

Could you bold the titles (`**Siphon (remote open)**`) and then instead of using a new line to separate the description, add two spaces at the end of the title link, and then have the description in the immediately following line.  See the python-awips notebooks below for example?

Also, I just noticed when previewing your page, that the most recent submit with the numpy and pandas links was not formatted properly either.  Would you mind changing those since you're working in this file already?

Reply:
Awesome, thanks!